app/views/CheckYourAnswersView.scala.html

```diff
@@ -26,9 +26,11 @@
     title = messages("checkYourAnswers.title")
     ) {
 
-    @components.heading("checkYourAnswers.heading")</h1>
+    @components.back_link()
+
+    @components.page_heading("amp.heading")
 
-    <p>@messages("checkYourAnswers.guidance")</p>
+    @components.heading("checkYourAnswers.heading")</h1>
```

Comment:
Is it ok to remove the closing h1 isn't needed as it is included in the component